docs/sources/flow/setup/install/binary.md

```diff
@@ -25,7 +25,7 @@ To download the Grafana Agent as a standalone binary, perform the following step
 
 1. Scroll down to the **Assets** section.
 
-1. Download the `grafana-agent-flow` zip file that matches your operating system and machine's architecture.
+1. Download the `grafana-agent` zip file that matches your operating system and machine's architecture.
```

Comment:
This is the "flow" section, so isn't "grafana-agent-flow" more correct? Or is the change because there is no "grafana-agent-flow-darwin" zip file?

Reply:
I think that's the reason; we only have `-flow` suffixed assets for deb/rpm and the windows installer.

Reply:
If we advise users to use `grafana-agent`, then we also would have to mention how to enable flow mode explicitly? Otherwise the instructions would be incomplete.

IMO we should just work to add the missing -flow binaries, since it seems like an oversight. I can't think of a reason why they we shouldn't have them.

Reply:
> If we advise users to use grafana-agent, then we also would have to mention how to enable flow mode explicitly? Otherwise the instructions would be incomplete.

I think that's covered in the "Next steps" part of the guide, and the "Start Grafana Agent" section.

> IMO we should just work to add the missing -flow binaries, since it seems like an oversight. I can't think of a reason why they we shouldn't have them.

The reason we initially added them as only as system packages was that, with system packages there was no way to run static mode and flow mode in parallel, so it was a way to offer users a workaround. On the other hand, with the binary, it's easy to do so (we just run it twice, with different AGENT_MODE variables).

We certainly _could_ add them as separate binaries, but I'm not sure if there's an immediate benefit here IMHO, what do you think?

Reply:
Just to verify things here. The steps to explicitly enable flow mode on the standalone binaries is included in the Start steps for the binary (the command includes `AGENT_MODE=flow`. This PR fixes up the documented instructions for the state as things are right now.

Reply:
Thanks Clayton, that sounds fine.